README.md

```diff
@@ -1,5 +1,7 @@
 # btest ... A minimal test framework
 
-This is a minimal test framework, extracted from [TeFLL](https://github.com/davidg238/fuzzy_logic) and now better aligned with the semantics of [GoogleTest](https://github.com/google/googletest).  
+A minimal test framework.
```

Comment:
Unless specified in the package.yaml, this is the description used for the package when publishing it.